### ISSUE TITLE
[Other] No VA.gov Experience Standard for the issue found. (00.00.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#59499]
#### VFS team name
MHV

#### VFS product name
My HealtheVet

#### VFS feature name
Secure Messaging

#### Point of Contact/Reviewers
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
No VA.gov Experience Standard for the issue found.

### Issue Details
On [/compose/](https://staging.va.gov/my-health/secure-messages/compose), the "Edit your contact list" modal contains a link out to the existing MHV application's contact list. That link opens in a new tab, but doesn't warn users about that behavior. That change of context can be disorienting for screen reader users who don't get the visual cues of the new tab opening, and can increase cognitive load across the board.

### VA.gov Experience Standard
[Category Number 00, Issue Number 00](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Platform Recommendation
Include a warning about the new tab behavior in the link text, eg. `<a ... href="..." target="_blank">Edit your contact list on the My HealtheVet website (opens in a new tab)</a>`. If that doesn't work for design purposes, then at minimum include the warning in an `aria-label` for screen reader users, eg. `<a ... href="..." target="_blank" aria-label="Edit your contact list on the My HealtheVet website (opens in a new tab)">Edit your contact list on the My HealtheVet website</a>`.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by lichellebain** [comment]
Jira ticket created:  https://jira.devops.va.gov/browse/MHV-46056
